File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/Samples/Tutorials/clr_fundamentals/samples/memorymanagement/usingcleanup/vb/UsingCleanup.vb
imports System
imports System.IO

class App 
   Shared Sub Main()
      Dim filename As String = "TemporaryFile.dat"

      Console.WriteLine("Creating File")
      Dim fs As New FileStream(filename, FileMode.Create)

      Dim data() As Byte = { 1, 2, 3, 4, 5, 6, 7, 8, 9 }
      Console.WriteLine("Writing to file")
      fs.Write(data, 0, data.Length)

      Console.WriteLine("Closing file")
      fs.Close()

      Console.WriteLine("Deleting file")
      File.Delete(filename)

      Console.WriteLine()
      Console.Write("Press Enter to close window...")
      Console.Read()
   End Sub
End Class

File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/Samples/Tutorials/clr_fundamentals/samples/memorymanagement/usingcleanup/cs/UsingCleanup.cs
using System;
using System.IO;

class App {
   public static void Main() {
      String filename = "TemporaryFile.dat";

      Console.WriteLine("Creating File");
      FileStream fs = new FileStream(filename, FileMode.Create);

      Byte[] data = { 1, 2, 3, 4, 5, 6, 7, 8, 9 };
      Console.WriteLine("Writing to file");
      fs.Write(data, 0, data.Length);

      Console.WriteLine("Closing file");
      fs.Close();

      Console.WriteLine("Deleting file");
      File.Delete(filename);

      Console.WriteLine();
      Console.Write("Press Enter to close window...");
      Console.Read();
   }
}